src/main/java/com/esri/samples/featurelayers/feature_collection_layer/FeatureCollectionLayerSample.java

```diff
@@ -0,0 +1,220 @@
+
+      // create feature collection and add to the map as a layer
+      FeatureCollection featureCollection = new FeatureCollection();
+      FeatureCollectionLayer featureCollectionLayer = new FeatureCollectionLayer(featureCollection);
+      map.getOperationalLayers().add(featureCollectionLayer);
+
+      // add point, line, and polygon geometry to feature collection
+      createPointTable(featureCollection);
+      createPolylineTable(featureCollection);
+      createPolygonTables(featureCollection);
+
+      // add the map view to stack pane
+      stackPane.getChildren().addAll(mapView);
+    } catch (Exception e) {
+      // on any error, display the stack trace.
+      e.printStackTrace();
+    }
+  }
+
+  /**
+   * Creates a Point Feature Collection Table with one Point and adds it to the Feature collection that was passed.
+   * 
+   * @param featureCollection
+   */
+  private void createPointTable(FeatureCollection featureCollection) {
+
+    // defines the schema for the geometry's attribute
+    List<Field> pointFields = new ArrayList<>();
+    pointFields.add(Field.createString("Place", "Place Name", 50));
+
+    // a feature collection table that creates point geometry
+    FeatureCollectionTable pointsTable = new FeatureCollectionTable(pointFields, GeometryType.POINT, WGS84);
+
+    // a default symbol used to display any features from the feature collection table that are not already assigned a
+    // symbol
+    SimpleMarkerSymbol markerSymbol = new SimpleMarkerSymbol(Style.TRIANGLE, 0xFFFF0000, 18);
+    SimpleRenderer renderer = new SimpleRenderer(markerSymbol);
+    pointsTable.setRenderer(renderer);
+
+    // add feature collection table to feature collection
+    featureCollection.getTables().add(pointsTable);
+
+    // create feature using the collection table by passing an attribute and geometry
+    Map<String, Object> attributes = new HashMap<>();
+    attributes.put(pointFields.get(0).getName(), "Current Location");
+    Point point = new Point(-79.497238, 8.849289, WGS84);
+    Feature addedFeature = pointsTable.createFeature(attributes, point);
+
+    // add feature to collection table
+    pointsTable.addFeatureAsync(addedFeature);
+  }
+
+  /**
+   * Creates a PolyLine Feature Collection Table with one PolyLine and adds it to the Feature collection that was passed.
+   * 
+   * @param featureCollection
+   */
+  private void createPolylineTable(FeatureCollection featureCollection) {
+
+    // defines the schema for the geometry's attribute
+    List<Field> polylineFields = new ArrayList<>();
+    polylineFields.add(Field.createString("Boundary", "Boundary Name", 50));
+
+    // a feature collection table that creates polyline geometry
+    FeatureCollectionTable polylineTable = new FeatureCollectionTable(polylineFields, GeometryType.POLYLINE, WGS84);
+
+    // a default symbol used to display any features from the feature collection table that are not already assigned a
+    // symbol

```

Comment:
How about "set a default symbol for features in the collection table"? The word "default" implies it will be replaced if a feature assigns their own symbol